puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/release/tfm-ror52-release.yaml

```diff
@@ -0,0 +1,15 @@
+---
+- job:
+    name: tfm-ror52-release
+    project-type: pipeline
+    concurrent: false
```

Comment:
duplicate with the `        disableConcurrentBuilds()` above?

Reply:
This version of it is more robust than using in the pipeline so I removed that reference.